client/my-sites/site-settings/cloudflare.js

```diff
@@ -30,32 +40,74 @@ const Cloudflare = () => {
 		<>
 			{ showCloudflare && (
 				<>
-					<SettingsSectionHeader title={ translate( 'Cloudflare CDN' ) } />
-					<Card>
+					{ ! sitePlan && <QuerySitePlans siteId={ siteId } /> }
+					<SettingsSectionHeader title={ translate( 'CDN' ) } />
+					<CompactCard>
 						<div className="site-settings__cloudflare">
+							<div className="site-settings__cloudflare-illustration">
+								<img src={ jetpackIllustration } alt="" />
+							</div>
 							<div className="site-settings__cloudflare-text">
+								<p className="site-settings__cloudflare-title">Jetpack Site Accelerator</p>
```

Comment:
Do we need to make this string available for translation even though it's a product name?